playbooks/ovs_2_ml2_cleanup_remnants.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+# a cleanup script to run after a host has been migrated from OVS to ml2 / linux bridge
+
+- hosts: all
+  serial: 10
```

Comment:
doesn't serial mean "do everything in parallel (per default behavior) but limit the batch size to 10?